cmd/pipeline/main.go

```diff
@@ -827,7 +827,6 @@ func main() {
 			cgroupsAPI.AddRoutes(orgs.Group("/:orgid/clustergroups"))
 
 			cRouter.GET("/nodepool-labels", nplsApi.GetNodepoolLabelSets)
-			cRouter.POST("/nodepool-labels", nplsApi.SetNodepoolLabelSets)
```

Comment:
Is this not used any more?

Reply:
it not used by UI as labels are updated in cluster update request